test/Test/Main.purs

```diff
@@ -641,6 +662,7 @@ main = do
   test_liftEff
 
   void $ launchAff do
+    test_lazy
```

Comment:
Can we move this test? They are ordered by requirements, so this should go after kill tests.